### ISSUE TITLE
[DevTools Bug]: shows "This page doesn't appear to be using React" for every react website on Firefox 115.0.2

### ISSUE THREAD

**Issue by anvesh-vanam** [facebook/react#27158]
### Website or app

all of them and also localhost. https://react.dev/ for example

### Repro steps

1. open any website made with React
2. Click on the extension icon

OS:
Windows 10

Browser:
Firefox 115.0.2 (64-bit)

DevTools:
4.27.8

### How often does this bug happen?

Every time

**Comment by hoxyq** [comment]
Hey @anvesh-vanam, thanks for opening this issue. 

1. Are you using device toolbar (ctrl + shift + m) with some selected device? This may change user agent and we currently aware of this problem.
2. Has this problem occurred after upgrading Firefox? Does it work with previous versions?

**Comment by anvesh-vanam** [comment]
@hoxyq  Hi. 
1.  i sometimes do. but it happens regardless of whether im using it or not.
2. i dont remember exactly but i think this has been happening for 2-3 weeks now, which seems to coincide closely with the 
115.0.2 release date. i am on 115.0.3 facing the same issue

**Comment by anvesh-vanam** [comment]
> Hey @anvesh-vanam for me reinstalling the the extension when noticing this behavior did the join. Had the same issue on Chrome a couple of times.

tried it multiple times. unfortunately didnt work.

**Comment by apicgg** [comment]
@anvesh-vanam Try upgrading Firefox to v.116.0.1. Is this issue only occurring on windows?

**Comment by JesseKoldewijn** [comment]
Can confirm that the before mentioned issue doesn't occur on the latest firefox dev-ed on win11. Personally haven't experienced this issue on any browser in linux yet.

**Comment by anvesh-vanam** [comment]
> @anvesh-vanam Try upgrading Firefox to v.116.0.1. Is this issue only occurring on windows?

i upgraded to 116.0.1. the issue is still there. yeah im only using windows 10

**Comment by JesseKoldewijn** [comment]
> 

Shouldn't matter all that much. Could you try the firefox developer edition to see if that one does work?

**Comment by anvesh-vanam** [comment]
> > 
> 
> Shouldn't matter all that much. Could you try the firefox developer edition to see if that one does work?

yeah i just installed developer edition and its working fine on that one

**Comment by apicgg** [comment]
@anvesh-vanam I am not able to reproduce this issue in my windows machine. Good to hear that it's working with dev edition of Firefox.

**Comment by JesseKoldewijn** [comment]
> 

Hey man, it seems to be a running issue that the dev tools sometimes break on older browser revisions. Experienced this with multiple releases over the past 2 years to be honest and either running dev/beta browsers or reinstalling the full browser itself usually does the job.

**Comment by coderdix24** [comment]
having the same issue in mac the firefox developer edition 117.0b8 (64-bit)

**Comment by henricook** [comment]
Also having this problem on 118.0.2 - did you ever find a fix?

**Comment by JesseKoldewijn** [comment]
> Also having this problem on 118.0.2 - did you ever find a fix?
> 
> 
> 
> !!window.React is false for me also

Have you tried reinstalling the extension?

**Comment by henricook** [comment]
> Have you tried reinstalling the extension?

Yes, no change.
